src/app/workspace/resource/resource-instance-form/select-ontology/select-ontology.component.html

```diff
@@ -1,12 +1,18 @@
 <span *ngIf="ontologiesMetadata !== undefined" [formGroup]="form">
     <mat-form-field *ngIf="ontologiesMetadata.ontologies.length === 1" class="large-field selected-ontology">
-        <mat-label>Select an ontology</mat-label>
+        <mat-label>Select a data model *</mat-label>
         <input matInput disabled value="{{ontologiesMetadata.ontologies[0].label}}" class="onto-label">
+        <mat-error *ngIf="form.controls.ontologies.hasError('required')">
```

Comment:
If this is automatically chosen for the user, do we ever encounter a scenario where this error message is needed?

Reply:
fixed in 3717fc5

Reply:
no you are right, I do not think that we will ever encounter such scenario

Reply:
fixed in ec83af5